src/pages/Cart/CartPage.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState, useEffect, useContext } from 'react';
+import { CartCard } from './CartCard/CartCard';
+import './CartPage.scss';
+import { useHistory } from 'react-router-dom';
+import {MyContext} from '../../App';
+import { GoBack } from '../../components/GoBack/GoBack';
+
+export const CartPage = () => {
+
+  const {cart, setCart} = useContext(MyContext);
+
+  const stringedCart = cart.map(item => JSON.stringify(item));
```

Comment:
why do you do this?

Reply:
This way I quickly get an array of non-duplicate products (phones)